### PR TITLE
Feature overview

### PR DESCRIPTION
**Pull request by spetebroodt**

### CODE REVIEW

File: user-guide/Reference/Feature_overview.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+| External user authentication via SAML - Okta | 10.3.0/10.3.2 |   |   |   |
+
+## Web apps
+
+|Feature | Minimum DataMiner version | Soft-launch option | License | Database |
+|---|---|---|---|---|
+| [Automatic backup of all dashboards and low-code apps during DMA upgrade or Web upgrade](xref:Upgrading_Downgrading_Webapps#automatic-backup) | 10.3.11/10.3.0 [CU8] |   |   |   |
+| [DataMiner web upgrades](xref:Upgrading_Downgrading_Webapps) | 10.3.0/10.3.3 |   |   |   |
+| Message throttling | 10.2.0/10.1.2 |   |   |   |
+| Template editor | 10.2.12 | ReportsAndDashboardsDynamicVisuals<br>ReportsAndDashboardsScheduler |   |   |
```
Comment (MariekeGO) [inline user-guide/Reference/Feature_overview.md:74]:
```suggestion
| Template Editor | 10.2.12 | ReportsAndDashboardsDynamicVisuals<br>ReportsAndDashboardsScheduler |   |   |
```

File: user-guide/Reference/Feature_overview.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---
+uid: Feature_overview
+---
+
+# Feature overview
+
+## Analytics
+
+|Feature | Minimum DataMiner version | Soft-launch option | License | Database |
```
Comment (MariekeGO) [inline user-guide/Reference/Feature_overview.md:9]: We should change the headers like this:
```suggestion
|Feature | Minimum DataMiner version | Minimum soft-launch version | Notes |
```
"Minimum DataMiner version" should contain "N/A" if the feature is only available in soft launch.

File: user-guide/Reference/Feature_overview.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+| Dashboards/Low-Code Apps: [*Stepper* component](xref:DashboardStepper) | DataMiner 10.3.10/10.4.0 <!-- [ID 37200] -->|
+| Dashboards/Low-Code Apps: [Template editor](xref:Template_Editor) | DataMiner web 10.4.1/10.5.0 <!-- [ID 34761] --><br>DataMiner 10.2.12 with the soft-launch options [ReportsAndDashboardsDynamicVisuals](xref:Overview_of_Soft_Launch_Options#reportsanddashboardsdynamicvisuals) and [ReportsAndDashboardsScheduler](xref:Overview_of_Soft_Launch_Options#reportsanddashboardsscheduler) |
+| Dashboards/Low-Code Apps: [*Text input* component](xref:DashboardTextInput) | DataMiner web 10.3.5/10.4.0 <!-- [ID 35902] -->|
+| Dashboards/Low-Code Apps: [Variables](xref:Variables) | DataMiner web 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 <!-- [ID 41039] [ID 41063] [ID 41132] -->|
+| Dashboards/Low-Code Apps: [*Timeline* component](xref:DashboardTimeline) | DataMiner web 10.4.1/10.5.0 <!-- [ID 34761] --><br>DataMiner 10.1.10 with [soft-launch option](xref:Overview_of_Soft_Launch_Options#reportsanddashboardsscheduler) <!-- [ID 37812] -->|
```
Comment (Robin-Debel) [inline user-guide/Reference/Feature_overview.md:56]: I believe with the soft launch flag this should be 10.2.12? https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=34761

**Review by Robin-Debel — COMMENTED** [review]
All Dashboards/LCA features checked

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**